### PR TITLE
Fix #68 Move popup to outside of the window when closing it

### LINKED ISSUE #68: popup layout and animation issues
The popup obscures the content below it even though it is hidden.

The [current implementation](https://github.com/tc39/beta/blob/65d61e221df72a92b2a79d4987135c9fd8e9243f/_sass/_popup.scss#L12-L16) for hiding it hides only the `label` using the `opacity`. The links/content underneath are still obscured by the popup preventing any interactions. The parent (`div.popup`) needs to be hidden instead of the label, which might require JS.

image

And I would also like to mention that the keyframe `moving` has visually no effect because while the 90% mark to move the `label` is reached the label would have an opacity of `0.1` which makes it nearly invisible.

image

### PR DESCRIPTION
**Pull request by othree**
This PR make sure the popup not cover any content of the page when it is closed by move it to outside of the window.

### CODE REVIEW

**Comment by littledan** [comment]
@mram What kind of feedback were you looking for from me? I don't have an opinion on the technical aspects of this PR.

File: _sass/_popup.scss [hunk 1]
```diff
@@ -6,26 +6,30 @@
 }
 
 input[type="checkbox"] + label {
```
Comment (jasonwilliams) [inline _sass/_popup.scss]: Should this style decleration be scoped within .popup?

File: _sass/_popup.scss [hunk 2]
```diff
@@ -6,26 +6,30 @@
...
-  position:relative;
+  position: absolute;
+  transition: 
+    opacity 0.3s ease-in-out,
+    transform 0.6s ease-in-out;
+  will-change: opacity, transform;
 }
 
 input[type="checkbox"]:checked + label {
```
Comment (jasonwilliams) [inline _sass/_popup.scss]: same here

File: _sass/_popup.scss (hunk 1 above)
Comment (r0mflip) [reply _sass/_popup.scss]: Better to be

**Comment by othree** [comment]
I added a new commit to make everything belongs to popup scoped in .popup

**Review by jasonwilliams — APPROVED** [review]

**Comment by jasonwilliams** [comment]
Thanks!

**Pull request merged**